.github/workflows/smoke-test.yml

```
@@ -199,7 +199,9 @@ jobs:
         # We only expect test coverage with all extra dependencies.
         # "polars" implies "scikit-learn": see setup.cfg.
         if: ${{ contains(matrix.extra-install, 'polars') && contains(matrix.python-version, '3.12') }}
-        run: coverage report | tee /tmp/coverage.txt
+        run: |
+          set -o pipefail
+          coverage report | tee /tmp/coverage.txt
```

Realized this can be much simpler!
```suggestion
          coverage report
          touch /tmp/coverage.txt
```
We don't actually need the report: We just use the artifact to confirm that it ran somewhere in the test matrix.